compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/InteractionUIView.uikit.kt

```diff
@@ -83,7 +83,7 @@ internal class InteractionUIView(
      * https://developer.apple.com/documentation/uikit/uiview/1622533-point
      */
     override fun pointInside(point: CValue<CGPoint>, withEvent: UIEvent?): Boolean {
-        val pointOffset = point.useContents { this.toDpOffset() }
+        val pointOffset = point.useContents { this.asDpOffset() }
```

Comment:
Since cinterop imports all C-structs as `CValue<SomeCStruct>`, perhaps it would be better to add `conversation` sugar to the `CValue` wrapper instead?

`fun CValue<CGPoint>.toDpOffset()`